### PR TITLE
Fm edits

### PR DESCRIPTION
**Pull request by fshmcallister**
First draft of SELinux - needs some heavy editing

### CODE REVIEW

File: _posts/selinux-vs-apparmor.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+# What is SELinux? Who is it for? How do I Use it?
+
+## Introduction
+
+Today we're going to talk about SELinux- Security Enhanced Linux. For the everyday layman SELinux is well above what most people or companies require, but is a good introduction into different ways to manage access control to your systems. More and more we give companies our precious data and the onus should be placed on developers to protect this data with growing cybersecurity threats. So we're going to show you how to implement SELinux for the security-conscious developer (and talk a little on AppArmor).
+
+Proceed with caution though- SELinux and AppArmor are MAC protocols (more on that later) and you run the risk of locking yourself out of your own system. If you're interested in learning a bit more about SELinux and MAC protocols, read on below. If you just want to jump straight into the bare-bones edition, skip straight ahead to the [tutorial](#tutorial) itself.
+
+**Required Skill Level**: Medium
+**Prerequisites**:
```
Comment (AlEsmail) [inline _posts/selinux-vs-apparmor.md]: Not displaying correctly

File: _posts/selinux-vs-apparmor.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+current: post
+cover: server-room.jpg
+title: What is SELinux and Why You Might Want it
+date: 2019-09-09 08:00:00
+category: tutorials
+author: Fiona McAllister
+tags: [tutorial]
+class: post-template
+---
+# What is SELinux? Who is it for? How do I Use it?
```
Comment (AlEsmail) [inline _posts/selinux-vs-apparmor.md]: Not displaying correctly

File: _posts/selinux-vs-apparmor.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+## Introduction
```
Comment (AlEsmail) [inline _posts/selinux-vs-apparmor.md]: Not displaying correctly

File: _posts/selinux-vs-apparmor.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+Today we're going to talk about SELinux- Security Enhanced Linux. For the everyday layman SELinux is well above what most people or companies require, but is a good introduction into different ways to manage access control to your systems. More and more we give companies our precious data and the onus should be placed on developers to protect this data with growing cybersecurity threats. So we're going to show you how to implement SELinux for the security-conscious developer (and talk a little on AppArmor).
+
+Proceed with caution though- SELinux and AppArmor are MAC protocols (more on that later) and you run the risk of locking yourself out of your own system. If you're interested in learning a bit more about SELinux and MAC protocols, read on below. If you just want to jump straight into the bare-bones edition, skip straight ahead to the [tutorial](#tutorial) itself.
+
+**Required Skill Level**: Medium
```
Comment (AlEsmail) [inline _posts/selinux-vs-apparmor.md]: Add 'Time to complete: X minutes' I'm not sure how long it actually takes but make something up.

File: _posts/selinux-vs-apparmor.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+## So what is SELinux and what is a MAC Model?
+
+SELinux is a US National Security Agency project aimed at improving security in the Linux kernel (more specifically, kernel 2.6.x). You will already be familiar with Discretionary Access Control as it is the system employed by most consumer Operating Systems. File permissions are determined by the creator/user, in Linux this is the 'Access Control List' - think about times where you have used `chmod` or `sudo` to assign read or write permissions.
+
+Mandatory Access Control, or MAC (not to be confused with Media Access Control) is different. Basically, the *operating system* determines access based on a security label (rules for access can be managed by a security officer- usually a single system administrator), not the user that created the file. For Linux, this system exists as SELinux. 
+
+The control is taken away from the user?! Why on Earth would you want that?! Many organisations deal with extremely sensitive data like military or governmen. Within a military organisation, some information is reserved for specific individuals with associated security labels (need to know, top secret, etc;).
```
Comment (AlEsmail) [inline _posts/selinux-vs-apparmor.md]: spelling: governmen = government

File: _posts/selinux-vs-apparmor.md (hunk 1 above)
Comment (vpetersson) [reply _posts/selinux-vs-apparmor.md]: That's expected. You can't preview the formatting like this.

File: _posts/selinux-vs-apparmor.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+## Finishing Up
+
+And that concludes our intro to SELinux. Play around with policies until you are comfortable, then you can ship SELinux to your own servers with confidence. SELinux essentially acts as a sandbox protecting your information from fradulent or corrupted access by external programs or daemons. As for AppArmor, it's just another MAC protocol that's used particularly for Ubuntu. It's easier to use as it deals with pathing rules; but SELinux is widely considered the more secure of the two. SELinux is better for those who are very familiar with Unix based systems, but AppArmor is another great introduction to MAC. 
+
+SELinux is a great way to implement security, but it is known for its bugs and disruptive mechanisms. Actual sandboxing is another alternative to protecting your kernel.
+
+Another way you can look to secure your devices- particularly if they are edge-based or use the net; is our very own agent here at WoTT. It's simple to set up and flexible, so why not head on over to our [getting started]({{site.url}}/documentation/getting-started) page and give it a go?
```
Comment (AlEsmail) [inline _posts/selinux-vs-apparmor.md]: Replace last paragraph with:

Good work getting to the end of this tutorial. You have taken a big step in securing your linux environment. There are additional steps you can take to improve security posture like CVE scanning, firewall management and service configuration. [WoTT](https://wott.io/documentation/getting-started) can help you do that with a lightweight agent that scans for vulnerabilities and provides clear recommended actions to fix those issues.

**Comment by fshmcallister** [comment]
Review please

**Pull request merged**